starlark/src/eval/def.rs

```diff
@@ -71,54 +78,186 @@ impl Def {
             stmts,
             captured_env: env,
             map,
-            local_names,
+            local_name_to_index,
         })
     }
 
-    fn collect_locals(stmt: &AstStatement, local_names: &mut HashSet<String>) {
+    fn collect_locals(stmt: &AstStatement, local_name_to_index: &mut HashMap<String, usize>) {
```

Comment:
```suggestion
    fn collect_locals(stmt: &AstStatement, local_names_to_index: &mut HashMap<String, usize>) {
```

Since there is several names.

Reply:
Names map 1-1 to indices. So it should be either "name to index" or "names to indices".